OnPod/Node/Common/PeripheralInterfaces/include/I2CInterface.h

```diff
@@ -0,0 +1,53 @@
+#ifndef H_I2CINTERFACE
+#define H_I2CINTERFACE
```

Comment:
Just to keep it consistent we can use FILENAME_H. Try I2CINTERFACE_H

Reply:
Went with discussed NODE_I2CINTERFACE_H